File: Lista de Exercícios (C)/C/Lista 2 - Casa/4.c
#include <stdio.h>

int main()
{
    float A, B, C;
    
    printf("Maior, menos e número do meio\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");
    
    printf("1º número: "); scanf("%f", &A);
    printf("2º número: "); scanf("%f", &B);
    printf("3º número: "); scanf("%f", &C);
    printf("\n");
    
    if (A>B && A>C) {
        printf("Maior número: %.2f\n", A);
        if (B>C) {
            printf("Número do meio: %.2f\n", B);
            printf("Menor número: %.2f\n", C);
        }
        else {
            printf("Número do meio: %.2f\n", C);
            printf("Menor número: %.2f\n", B);
        }
    }
    else if (B>A && B>C) {
        printf("Maior número: %.2f\n", B);
        if (A>C) {
            printf("Número do meio: %.2f\n", A);
            printf("Menor número: %.2f\n", C);
        }
        else {
            printf("Número do meio: %.2f\n", C);
            printf("Menor número: %.2f\n", A);
        }
    }
    else {
        printf("Maior número: %.2f\n", C);
        if (B>A) {
            printf("Número do meio: %.2f\n", B);
            printf("Menor número: %.2f\n", A);
        }
        else {
            printf("Número do meio: %.2f\n", A);
            printf("Menor número: %.2f\n", B);
        }
    }
}

File: Listas de Exercícios/Listas de Exercícios (Python)/Python/Lista 2 - Casa/4.py
print("Maior, menor e número do meio\n")

print("--Valores decimais devem ser digitados com (.)!--")

A=float(input("1º número: "))
B=float(input("2º número: "))
C=float(input("3º número: "))
print("")

if (A>B and A>C):
    print(f"Maior número: {A}")
    if (B>C):
        print(f"Número do meio: {B}")
        print(f"Menor número: {C}")
    else:
        print(f"Número do meio: {C}")
        print(f"Menor Número: {B}")

if (B>C and B>A):
    print(f"Maior número: {B}")
    if (C>A):
        print(f"Número do meio: {C}")
        print(f"Menor número: {A}")
    else:
        print(f"Número do meio: {A}")
        print(f"Menor número: {C}")

if (C>A and C>B):
    print(f"Maior número: {C}")
    if (B>A):
        print(f"Número do meio: {B}")
        print(f"Menor número: {A}")
    else:
        print(f"Número do meio: {A}")
        print(f"Menor número: {B}")

print("")